app/src/main/java/org/wikipedia/descriptions/DescriptionEditTutorialFragment.kt

```diff
@@ -32,15 +32,20 @@ class DescriptionEditTutorialFragment : OnboardingFragment() {
         }
 
         override fun createFragment(position: Int): Fragment {
-            return ItemFragment().apply { arguments = bundleOf(ARG_POSITION to position) }
+            return ItemFragment(showAIOnBoarding).apply { arguments = bundleOf(ARG_POSITION to position) }
         }
     }
 
-    class ItemFragment : Fragment() {
+    class ItemFragment(private val showAIOnBoarding: Boolean) : Fragment() {
```

Comment:
Incorrect usage of fragment parameters. Parameters must be passed into a fragment using "arguments" which are given to a fragment using a "Bundle". (similar to how `position` is passed into the fragment, above)

If you do not provide a no-argument constructor, the app will crash when recreating the fragment, e.g. upon screen rotation, as you might have observed while testing.

Reply:
Used the parent fragment argument, since it is not position specific..